### DISCUSSION TITLE
Subí el formulario de las katas, con mi correo de innovacción. Entré al teams de Mision Front-End pero no me llegó ningún correo.

### DISCUSSION THREAD

**Discussion by CristopherSandoval** [LaunchX-InnovaccionVirtual/CursoIntroPython#317] [Q&A] [answered]
Hola! Muy buenas tardes/noches/días!
Tengo una duda, acabo de subir las katas como se explicó en la sesión y en el proceso de entrega!
Inmediatamente se me notificó que me habían añadido a un nuevo grupo de teams: Misión Frontend - JS... pero no me llegó ningún correo de confirmación :( 
Alguien sabe si tardan o son de inmediato?

**Comment by gersondlarosa** [comment]
Yo igual subí mis katas hoy pero no me llego ningún correo

**Reply by CristopherSandoval** [reply]
Pero tú igual que yo ya estás dentro del equipo de teams de Front-End?

**Comment by ArielCanedo** [comment, marked as answer]
Si estás dentro del grupo front end no hay problema
Saludos!!

**Reply by CristopherSandoval** [reply]
Excelente. Muchísimas gracias!
